### ISSUE TITLE
Search wrapping off.

### ISSUE THREAD

**Issue by krompus** [5digits/dactyl#146]
I would very much appreciate the ability to toggle the search wrapping off, so that it does not loop to the opposite end of the page. Is this already possible with a config setting?

**Comment by phanimahesh** [comment]
The equivalent vim options are `set [no]wrapscan` to set and `set wrapscan!` to toggle.
They are not recognized by pentadactyl.

**Comment by krompus** [comment]
Yep, I already have it in vim, and I've become addicted to it. :)

Not a big deal by any means, but it would be nice.
